website_sale_delivery_excluded_vat_label/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2017- Houssine BAKKALI - Coop IT Easy SCRLfs
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "Website Sale Delivery Excuded VAT Label",
+    "version": "12.0.1.0.0",
+    "category": "E-commerce",
+    "sequence": 95,
+    "author": "Coop IT Easy SCRLfs",
+    "license": "AGPL-3",
```

Comment:
maybe add the coopiteasy website key

Reply:
done.
But now that I see it: what about `"sequence": 95,` ?

Reply:
you can remove it